packages/salesforcedx-vscode-core/src/services/sdr/componentSetUtils.ts

```diff
@@ -54,16 +54,22 @@ export async function setSourceApiVersion(componentSet: ComponentSet): Promise<v
 
   // Next, if it still is not set, set it to the highest API version supported by the target org.
   if (!sourceApiVersion) {
-    const orgApiVersion = await getOrgApiVersion();
+    const orgApiVersion = await componentSetUtils.getOrgApiVersion();
```

Comment:
Call `componentSetUtils.getOrgApiVersion()` now so that `getOrgApiVersion()` can be mocked in unit tests.